### PR TITLE
Updated Adobe Reader DC package

### PR DESCRIPTION
**Pull request by proudcanadianeh**
Added uninstall script, check for current version already installed, and
support for passing through LCID. (Although installer seems to use the
windows LCID as default anyways)

### CODE REVIEW

**Comment by gep13** [comment]
@proudcanadianeh so, I don't know too much about the change that you are making, I am assuming that this is something you have discussed with @ferventcoder, but one thing I would say is...

This package is one of Rob's automatic packages, which uses a ketarin instance, to populate variables within the various files, before the nupkg is created.  As an example, here:

https://github.com/ferventcoder/chocolatey-packages/pull/64/files#diff-3803df9cf6dc58779f7d51c72242bbfdL2

The ```{{PackageVersion}} property is one that ketarin will look for and replace as part of the automatic creation of the nupkg.  By putting the version number in directly as you have, the automatic package creation will no longer work correctly.

As a minimum, I think you are going to need to put any ```{{ }}```'s back in place.

Does that make sense?

File: automatic/adobereader/tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -1,90 +1,94 @@
...
+      Write-Debug "No Package Parameters Passed in"
+  }
+
+  $scriptDir = $(Split-Path -parent $MyInvocation.MyCommand.Definition)
+  Import-Module (Join-Path $scriptDir 'checkinstall.ps1')
+  #If no Language ID passed through, use the one from Windows
+  #LCID table can be found at http://msdn.microsoft.com/es-es/goglobal/bb964664.aspx to manually pass through the desired language. 
+   if (!$lcid){ 
+    $lcid = (Get-Culture).LCID
+    echo "LCID set to $lcid"
```
Comment (ferventcoder) [inline automatic/adobereader/tools/chocolateyInstall.ps1]: Very nice.

File: automatic/adobereader/tools/chocolateyInstall.ps1 [hunk 2]
```diff
@@ -1,90 +1,94 @@
...
+  #LCID table can be found at http://msdn.microsoft.com/es-es/goglobal/bb964664.aspx to manually pass through the desired language. 
+   if (!$lcid){ 
+    $lcid = (Get-Culture).LCID
+    echo "LCID set to $lcid"  
+   }
+
+  
+  # Find download URLs at http://www.adobe.com/support/downloads/product.jsp?platform=windows&product=10
+  
+  $url = 'http://ardownload.adobe.com/pub/adobe/reader/win/AcrobatDC/1500720033/AcroRdrDC1500720033_MUI.exe'
```
Comment (ferventcoder) [inline automatic/adobereader/tools/chocolateyInstall.ps1]: Url will need to go back to `{{DownloadUrl}}` - but @gep13 already mentioned that. I can probably do the fixup for you.

File: automatic/adobereader/tools/chocolateyInstall.ps1 [hunk 3]
```diff
@@ -1,90 +1,94 @@
...
+          echo "LCID set to $lcid"          
+      } 
+
+
+  } else {
+      Write-Debug "No Package Parameters Passed in"
+  }
+
+  $scriptDir = $(Split-Path -parent $MyInvocation.MyCommand.Definition)
+  Import-Module (Join-Path $scriptDir 'checkinstall.ps1')
```
Comment (ferventcoder) [inline automatic/adobereader/tools/chocolateyInstall.ps1]: checkInstall.ps1 appears to be missing.

File: automatic/adobereader/tools/chocolateyInstall.ps1 [hunk 4]
```diff
@@ -1,90 +1,94 @@
...
+  $installArgs = "/sALL /rs /sl $lcid"
+  
+ #Kill the Adobe Updater AdobeARM.exe
+  
+  Stop-Process -processname AdobeARM -ErrorAction SilentlyContinue
+
+  #Check for current version installed
+
+  #Check for OS bitness
+if ((Get-WmiObject Win32_OperatingSystem).OSArchitecture -match "64-bit"){
```
Comment (ferventcoder) [inline automatic/adobereader/tools/chocolateyInstall.ps1]: We have something that already does this `Get-ProcessorBits` - horribly named but it's about the OS bitness. https://github.com/chocolatey/choco/wiki/HelpersReference#complete-list-alphabetical-order

**Comment by proudcanadianeh** [comment]
Ok, just uploaded a new version with a number of changes that should be more friendly towards being automated. In this new version, all that needs to be modified is the $url and $version within the common.ps1

When I get a chance I am going to try and play a bit with the auto creation, in the meantime @ferventcoder any chance you could help me out just to get this package out there and going (As the AdobeReader package is now as of today 1 month behind)

**Comment by ferventcoder** [comment]
Closed in favor of #64 - thanks for your contribution though!

**Pull request closed without merging**